### PR TITLE
Added a function that creates strings

### PR DESCRIPTION
**Pull request by artheadsweden**
@WokieManWassbergIsKing can you please check this?

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,7 +1,13 @@
+import random
+
```
Comment (WokieManWassbergIsKing) [inline main.py:2]: Pep-8 says two blank lines here

File: main.py [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+def the_func(name):
+    return f"Yeah {name}"
+
```
Comment (WokieManWassbergIsKing) [inline main.py:6]: ... and here

**Review by WokieManWassbergIsKing — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (artheadsweden) [reply main.py:2]: Fixed this

File: main.py (hunk 2 above)
Comment (artheadsweden) [reply main.py:6]: and this

**Review by WokieManWassbergIsKing — APPROVED** [review]

**Pull request merged**